ballot/models.py

```diff
@@ -52,6 +52,10 @@ def __str__(self):
 class BallotItemResponse(models.Model):
     """
     YES/NO to a referendum, or a candidate.
+
+    This is a *conceptually* abstract class, but exists
+    as a Django model so that finance can point to propositions
+    and candidates equally.
     """
```

Comment:
I like Selection :)

> On Jan 14, 2016, at 12:49 PM, Ben Cipollini <notifications@github.com> wrote:
> 
> In ballot/models.py <https://github.com/caciviclab/disclosure-backend/pull/134#discussion_r49782860>:
> 
> > @@ -52,6 +52,10 @@ def __str__(self):
> >  class BallotItemResponse(models.Model):
> >      """
> >      YES/NO to a referendum, or a candidate.
> > +
> > +    This is a *conceptually* abstract class, but exists
> > +    as a Django model so that finance can point to propositions
> > +    and candidates equally.
> >      """
> Looks like the latest spec changed things a bit; it's called BallotSelection now: 
> https://github.com/votinginfoproject/vip-specification/blob/vip51/vip_spec.xsd#L180 <https://github.com/votinginfoproject/vip-specification/blob/vip51/vip_spec.xsd#L180>
> —
> Reply to this email directly or view it on GitHub <https://github.com/caciviclab/disclosure-backend/pull/134/files#r49782860>.
>